### DISCUSSION TITLE
Lifefitness GX longshot??! (lever type resistance)

### DISCUSSION THREAD

**Discussion by bsharp77** [doudar/SmartSpin2k#408] [General]
Hi guys,

Just found the amazing smartspin - a device I've been dreaming about in my head for years!!

I currently use a Lifefitness GX spin bike with Garmin Rally power pedals.

I chose the GX as its one of the few spin bikes that shows a digital readout of resistance level, making it easier for me when manually changing resistance...however it looks like its one of the few spin bikes that doesn't use a traditional knob which means its incompatible with the smartspin.

Would there be any way of adapting the smartspin to work with the Lifefitness GX"lever" type (the Keiser M3 has a very similar lever).

Failing that, my last resort would be to purchase another spin bike - I could possibly get my hands on a Matrix Tomahawk S-Series if anyone knows if this would be compatible? 

I'd dearly love it to get something to work with the Lifefitness GX if possible as it is an amazing spin bike - I'd be more than happy to work with anyone to develop this - my skills are limited, but my enthusiasm is high! :-)

Thanks a lot,
Ryan

**Comment by eMadman** [comment]
hoooboy, that would be fun but way outside the scope of the case design we have today.  If you're comfortable with CAD, all of our designs are available on the repo and you can try to make your own, but the lever mechanism isn't something we'll be able to adapt the existing designs to.  I understand why you'd want to adapt it for this bike,.  My condo's gym has some life fitness spin bikes and they really are a pleasure to ride.

Since your goals seem to be a quality bike that can stand up to years of abuse AND you have some fantastic pedals... I'd suggest looking at the Bowflex C6/Schwinn IC4.  They go on steep discounts often, have a mounting mechanism available, and the build quality is pretty impressive.

I have a mounting solution for the Life Fitness IC7 as well if you want to go even higher end.

**Reply by bsharp77** [reply]
Hi eMadman! Thanks very much for the swift response. 
Ah I totally understand - it was worth asking I suppose.
I love using a spin bike for training so much - there are just so many advantages for me.

Moving on from the Lifefitness, as mentioned I could pick up a Matrix Tomahawk S-Series at a reasonable cost from a refurbished gym bike seller close by here in N.Ireland, but would need to check with you if you think this would be compatible as it would be the sole reason for purchase!
![Screenshot 2022-11-29 at 15 46 25](https://user-images.githubusercontent.com/119433368/204577231-fe831ab2-3ec1-402f-9d98-9281e9fc0d3e.png)

The IC7 would be a dream solution, but once up in that sort of territory, the Kickr bike comes into play!

**Reply by bsharp77** [reply]
Another reason that makes this Matrix very appealing is the road style handlebars which is a bonus

**Comment by doudar** [comment]
It sure looks like it should be compatible. Is it magnetic or friction? (Magnetic is better) 

We’d have to make a new insert for it but that’s not a problem. @eMadman , what do you think?

**Reply by eMadman** [reply]
It'll likely need an ultrashort case.  Measure the distance from center of knob to the front tube - if you have 81mm or more of clearance, the direct mount would be suitable.  We have another case which works with 49mm of clearance but mounting with it is less rigid

I can't tell from the picture, but the knob will need some detents on it for the knob insert to have something to grab onto and turn.

This green line identifies the measurement i'll need
![image](https://user-images.githubusercontent.com/24726844/204597779-d2b9b022-b649-4da5-9efa-e6a1a409e7fb.png)
